CMakeLists.txt

```diff
@@ -91,23 +91,27 @@ add_custom_target(get_git_hash ALL
   DEPENDS ${PROJECT_BINARY_DIR}/lib/d3_version.h
 )
 
-if(UNIX)
+if(UNIX AND NOT HOST_TOOLS_ONLY)
```

Comment:
What is `HOST_TOOLS_ONLY`? I could not find any doc

Reply:
this is an external define i set in the android build so that it would skip configuring SDL for the host tools. this may be redundant now that SDL will be provided by vcpkg - i'll experiment to see if it configures/builds sdl regardless of whether i `find_package()` it.

if i do keep this "skip all configuration except for HogMaker" var, is there a more cmake-idiomatic naming convention to use?